roles/phpmyadmin/tasks/main.yml

```diff
@@ -0,0 +1,45 @@
+---
+  - name: "copy template files for docker build"
+    template:
+      src: "{{ item }}"
+      dest: '{{ build_path }}/{{ item | basename | regex_replace("\.j2$", "") }}'
+      owner: root
+      group: root
+      mode: "0744"
+      force: yes
+    with_fileglob:
+      - ../templates/build/*.j2
+
+  - name: "copy ssl template files for docker build"
+    template:
+      src: "{{ item }}"
+      dest: '{{ build_path }}/{{ item | basename | regex_replace("\.j2$", "") }}'
+      owner: root
+      group: root
+      mode: "0744"
+      force: yes
+    with_fileglob:
+      - ../templates/build/ssl/*.j2
+    when: myadmin_url is defined
```

Comment:
I think this task can be combined with the previous task since they're both copy templates from src to dest.

Reply:
I separated them out because they are only needed if you are setting up an https connection